apps/ledger-live-desktop/jest.config.js

```diff
@@ -72,10 +72,14 @@ module.exports = {
     "!src/**/__integration__/**",
     "!src/**/__tests__/**",
   ],
-  coverageReporters: ["json", "lcov", "json-summary"],
+  collectCoverage: true,
```

Comment:
I think that we might not want to collect coverage as default. When we develop a feature we just want to replay the test quickly. And we can control coverage collection through the cli. Is there a reason to force it in the config?

Reply:
yes ! just forgotten these lines